### PR TITLE
Enhancement: Added code to Jenkinsfile to make changes to the develop…

### PR DESCRIPTION
**Pull request by KevinFHartmann**
…ment branch trigger a deployment build with the 'dev' environment.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -76,6 +76,16 @@ spec:
             }
         }
 
+        stage('Deploy to argo-dev') {
+            when { branch 'development' }
```
Comment (blabadi) [inline Jenkinsfile]: develop

**Comment by blabadi** [comment]
just one more thing I forogt can you please tag latest and edge same as this : https://github.com/icgc-argo/argo-clinical/blob/master/Jenkinsfile.groovy

File: Jenkinsfile [hunk 2]
```diff
@@ -76,9 +76,42 @@ spec:
...
+                        sh 'docker login -u $USERNAME -p $PASSWORD'
+                    }
+
+                    // the network=host needed to download dependencies using the host network (since we are inside 'docker'
+                    // container)
+                    sh "docker build --network=host -f Dockerfile . -t icgcargo/program-service:latest"
+                    sh "docker push icgcargo/program-service:latest"
+                    withCredentials([usernamePassword(credentialsId: 'argoGithub', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')]) {
+                        sh "git tag ${commit}"
+                        sh "git push https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/icgc-argo/program-service --tags"
```
Comment (blabadi) [inline Jenkinsfile]: this is missing a closing bracket

Comment (KevinFHartmann) [reply Jenkinsfile]: Thanks! :-)

**Pull request merged**